src/coreclr/src/debug/createdump/createdumpunix.cpp

```diff
@@ -19,6 +19,8 @@ CreateDump(const char* dumpPath, int pid, MINIDUMP_TYPE minidumpType)
     {
         goto exit;
     }
+    printf("Process %d %s\n", crashInfo->Pid(), crashInfo->Name().c_str());
```

Comment:
Should this be a print, or is it meant to be a trace?

Reply:
I could go either way on this.  createdump already printfs other things like the dump path and other stats.